tutorials/getting-started-with-flow/05__connecting-the-form-to-the-app/content.adoc

```diff
@@ -3,17 +3,19 @@
 
 :title: Part 5 - Connecting the Form to the Application
 :author: Vaadin
-:description: Learn how to communicate between components in a Vaadin application
+:description: Learn how to communicate between components in a Vaadin Flow application
 :tags: Flow, Java
 :imagesdir: ./images
 :linkattrs:
 
-In this part of the tutorial, you'll learn how to connect the `CustomerForm` component (implemented in the previous step) to the application. This includes showing and hiding the form when a row is selected or deselected in the `Grid`, making the *Save* and *Delete* buttons work, and adding a _create new customer_ option.
+This part of the tutorial shows you how to connect the `CustomerForm` component (implemented in the previous step) to the application. 
 
-== Showing and Hiding the Form
+== Showing and Hiding the Form Data
```

Comment:
@ruthstocks "Form Data" sounds wrong here as well. Can you fix it, please?

Reply:
We are hiding the whole form, so that fields and buttons are not visible when the for is not active. So I think the original sounds better.

Reply:
Reverting now. Apologies for the delay